### DISCUSSION TITLE
Education supply page problem

### DISCUSSION THREAD

**Discussion by SamBianYiLin** [community/community#138081] [Actions]
In this page,when I want to switch my camera to back camera,the page will become blank and I cannot take pictures and submit my license. This troubled me a lot,it caused that I cannot get the student identification.

**Comment by VadymMakohon** [comment]
To assist with the issue of the page going blank when switching to the back camera, you can follow these steps:

Check Browser Permissions: Ensure the browser has permission to access your camera, and it is set to use the correct (back) camera.

Update the Browser: Make sure you are using the latest version of the browser. Outdated browsers can cause compatibility issues.

Try a Different Browser: If the issue persists, try switching to another browser (e.g., from Chrome to Firefox).

Clear Cache and Cookies: Sometimes, clearing the browser cache and cookies can resolve display issues.

Check for Errors: Right-click on the page, select "Inspect", and check the Console tab for any errors related to the camera.

Test on Another Device: Try accessing the page from a different device to see if the issue is specific to your current one.

Contact Support: If none of the above works, report the issue to the support team of the website, providing details of the problem.

These steps should help resolve or identify the issue.
